samples/notebooks/python/05-using-the-planner.ipynb

```diff
@@ -1,203 +1,299 @@
+    "\"\"\"\n",
+    "shakespeareFunction = kernel.create_semantic_function(sk_prompt, \"shakespeare\", \"ShakespeareSkill\",\n",
+    "                                                      max_tokens=1000, temperature=0.8)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "305743f5",
+   "metadata": {},
+   "source": [
+    "Let's update our ask using this new skill."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "bfd23ac4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ask = \"\"\"Tomorrow is Valentine's day. I need to come up with a few date ideas.\n",
+    " She likes Shakespeare so write using his style. E-mail these ideas to my significant other\"\"\"\n",
+    "\n",
+    "new_plan = await planner.create_plan_async(ask, kernel)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "b09c4834",
+   "metadata": {},
+   "outputs": [
     {
-      "cell_type": "code",
-      "execution_count": null,
-      "id": "cf30b5da",
-      "metadata": {},
-      "outputs": [],
-      "source": [
-        "# TODO: implement in Python"
-      ]
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "    {\n",
+      "        \"input\": \"Valentine's Day Date Ideas\",\n",
+      "        \"subtasks\": [\n",
+      "            {\"function\": \"WriterSkill.Brainstorm\"},\n",
+      "            {\"function\": \"ShakespeareSkill.shakespeare\"},\n",
+      "            {\"function\": \"WriterSkill.EmailTo\", \"args\": {\"recipient\": \"significant_other\"}}\n",
+      "        ]\n",
+      "    }\n"
+     ]
     }
-  ],
-  "metadata": {
-    "kernelspec": {
-      "display_name": "Python 3 (ipykernel)",
-      "language": "python",
-      "name": "python3"
-    },
-    "language_info": {
-      "codemirror_mode": {
-        "name": "ipython",
-        "version": 3
-      },
-      "file_extension": ".py",
-      "mimetype": "text/x-python",
-      "name": "python",
-      "nbconvert_exporter": "python",
-      "pygments_lexer": "ipython3",
-      "version": "3.10.10"
+   ],
+   "source": [
+    "print(new_plan.generated_plan)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5e6ac56",
+   "metadata": {},
+   "source": [
+    "### Executing the plan\n",
+    "\n",
+    "Now that we have a plan, let's try to execute it! The Planner has a function called `execute_plan`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "cf30b5da",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = await planner.execute_plan_async(new_plan, kernel)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "eb9727c0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Hi there,\n",
+      "\n",
+      "Are you looking for a romantic activity to do with your special someone? Here are some ideas that might be fun: \n",
```

Comment:
nit: This example doesn't really sound like Shakespeare to me. Also, it's a weird start to an email to your significant other.  I say this because I wasn't convinced the planner was working properly, so I ran it with some print statements to make sure each function was running.